usvm-python/usvm-python-main/build.gradle.kts

```diff
@@ -0,0 +1,35 @@
+import usvmpython.USVM_PYTHON_ANNOTATIONS_MODULE
+import usvmpython.USVM_PYTHON_COMMONS_MODULE
+import usvmpython.getGeneratedHeadersPath
+import usvmpython.tasks.generateJNIForCPythonAdapterTask
+
+plugins {
+    id("usvm.kotlin-conventions")
+}
+
+val headerPath = getGeneratedHeadersPath()
+
+tasks.compileJava {
+    // to suppress "No processor claimed any of these annotations: org.jetbrains.annotations.Nullable,org.jetbrains.annotations.NotNull"
+    options.compilerArgs.add("-Xlint:-processing")
+    options.compilerArgs.add("-AheaderPath=${headerPath.canonicalPath}")
+    outputs.dirs(headerPath)
+}
+
+tasks.build {
+    doLast {
+        generateJNIForCPythonAdapterTask()
+    }
+}
+
+dependencies {
+    implementation(project(":usvm-core"))
+    implementation(project(mapOf("path" to ":$USVM_PYTHON_ANNOTATIONS_MODULE")))
```

Comment:
Why do we need `"path"` here?

Reply:
Actually, this line is generated by IDEA. Without it, classes from that module were not resolved for some reason.

Reply:
On my machine everything resolves, but OK